node/src/service.rs

```diff
@@ -25,6 +27,13 @@ type FullClient = sc_service::TFullClient<Block, RuntimeApi, Executor>;
 type FullBackend = sc_service::TFullBackend<Block>;
 type FullSelectChain = sc_consensus::LongestChain<FullBackend, Block>;
 
+fn remote_keystore(_url: &String) -> Result<Arc<LocalKeystore>, &'static str> {
```

Comment:
Do we simply not support any remote keystore for the moment ?
This function is called later in `new_full()`, is there a point to calling it there ?

Reply:
No, there isn't need to call, will remove